chainer/utils/experimental.py

```diff
@@ -0,0 +1,107 @@
+import chainer
+import inspect
+import warnings
+
+
+def experimental(api_name=None):
```

Comment:
To be honest, I did not imagine such a usage, but I think it is better to support it. The most straightforward way is to check the argument in the function and call `experimental`

```python
def f(x, y, experimental_argument=None):
    if experimental_argument is not None:
        experimental()
```

I come to think we may have to change the warning message to support such usage.